moveit_core/kinematics_base/include/moveit/kinematics_base/kinematics_base.h

```diff
@@ -53,12 +52,15 @@ namespace core
 MOVEIT_CLASS_FORWARD(JointModelGroup)
 MOVEIT_CLASS_FORWARD(RobotState)
 MOVEIT_CLASS_FORWARD(RobotModel)
-}
-}
+}  // namespace core
+}  // namespace moveit
 
 /** @brief API for forward and inverse kinematics */
 namespace kinematics
 {
+// Logger
+rclcpp::Logger logger_kinematics_base = rclcpp::get_logger("kinematics_base");
```

Comment:
please let's stick with ``logger``

(or ``static const rclcpp::Logger LOGGER``)

Reply:
When using the form you recommend:
```bash
/Users/victor/ros2_moveit_ws/src/moveit2/moveit_core/kinematics_base/include/moveit/kinematics_base/kinematics_base.h:317:5: error: static_assert failed due to requirement '::std::is_same<typename std::remove_reference<decltype(kinematics::LOGGER)>::type, typename ::rclcpp::Logger>::value' "First argument to logging macros must be an rclcpp::Logger"
    RCLCPP_ERROR(kinematics::LOGGER, "This kinematic solver does not support searchPositionIK with multiple poses");
```

I didn't dive very deep into the problem and instead partially followed your comment at 8b4f35c57334d4d18025919e336b62e0daf80b45.